Code/Source/RobotControl/ControlConfiguration.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#pragma once
```

Comment:
The `#pragma once` in a cpp file will cause a compiler warning which will turn into a compile error due to `-Werror`

Reply:
@pijaro